plugin/src/test/java/io/jenkins/plugins/analysis/warnings/groovy/ParserConfigurationTest.java

```diff
@@ -19,14 +20,14 @@
     private static final List<GroovyParser> PARSERS = Collections.singletonList(mock(GroovyParser.class));
 
     @Test
-    void shouldHaveNoRootFoldersWhenCreated() {
+    void shouldHaveNoParsersWhenCreated() {
```

Comment:
I've renamed this test case to better describe what it's testing.
(I suspect there was some copy/paste of test names when this class was originally created)